filament/src/Texture.cpp

```diff
@@ -181,8 +181,31 @@ void FTexture::setExternalStream(FEngine& engine, FStream* stream) noexcept {
 
 void FTexture::generateMipmaps(FEngine& engine) const noexcept {
     if ((mTarget == Sampler::SAMPLER_2D || mTarget == Sampler::SAMPLER_CUBEMAP)
-            && mLevels > 1) {
-        engine.getDriverApi().generateMipmaps(mHandle);
+            && mLevels > 1 && (mWidth > 1 || mHeight > 1)) {
```

Comment:
We should only generate the mipmaps when the texture format is renderable.

Reply:
Ok, added a commit to the PR for this.